docs/interactive-pause.md

```diff
@@ -0,0 +1,58 @@
+# Interactive Pause
+
+It’s hard to write a complete test at once. You will need to try different commands with different arguments before you find a correct path.
+
+Since Codeception 3.0 you can pause execution in any point and enter interactive shell where you will be able to try commands in action. 
+
+Now this `Interactive Pause` feature is available in MFTF and all you need to do is to set `ENABLE_PAUSE=true` in `.env`.
+
+Check it out at [Codeception website][] for documentation and a video to see `Interactive Pause` in action.
+ 
+In short, when a test gets to `$I->pause()` step, it stops and shows a console where you can try all available commands with auto-completion, stash commands, and save screenshot, etc. 
+
+## Generation Time
+
+A `<pause>` action in xml will always be generated into php regardless if `ENABLE_PAUSE=true` is set or not. 
+However, when `ENABLE_PAUSE=true` is set, an additional`pause()` action will be generated in `_failed()` hook for a test,
```

Comment:
I see your intention here. Do we want to separate the docs into Generation/ Execution time?

Since pause does not change user experience for test generation, would it be better to have a section  called `usage` under `Interactive pause` secton instead, stating how to pause a test run, and about how test failures will also pause execution with examples?